src/index.js

```diff
@@ -25,7 +25,7 @@ function getLastDayOf(day, month, year) {
   return new Date(Date.parse(`${month}/${lastOfDay}/${year} GMT`));
 }
 
-function allFederalHolidaysForYear(year = (new Date().getFullYear())) {
+function allFederalHolidaysForYear(year = (new Date().getFullYear()), federalReserveMode = false) {
```

Comment:
Instead of making a Federal Reserve-specific option, what do you think about a more general option for shifting dates forward or back if they fall on the weekend?  E.g.,

```js
allFederalHolidaysForYear(year = new Date().getFullYear(), { shiftSaturday: true, shiftSunday: true } = {}) {
  ...
  if (dow === 0 && shiftSunday ) {
    ...
  } else if (dow === 6 && shiftSaturday) {
    ...
  }
}
```

That way folks could modify the behavior based on other arrangements of shifting dates as well.

Reply:
Works for me. 👍